src/EVault/modules/Governance.sol

```diff
@@ -236,7 +280,19 @@ abstract contract GovernanceModule is IGovernance, Base, BalanceUtils, BorrowUti
         uint16 originalLTV = getLTV(collateral, true).toUint16();
         vaultStorage.ltvLookup[collateral].clear();
 
-        emit GovSetLTV(collateral, 0, 0, 0, originalLTV);
+        emit GovSetLTV(collateral, 0, 0, originalLTV, 0, 0);
+    }
+
+    /// @inheritdoc IGovernance
+    function setMaxLiquidationDiscount(uint16 newDiscount) public virtual nonReentrant governorOnly {
+        vaultStorage.maxLiquidationDiscount = newDiscount.toConfigAmount();
```

Comment:
Should this function have any lower/upper bounds for the `newDiscount` value, or do you want to allow the Governor to input anything up to `10_000` (internal check done by `.toConfigAmount()`)?

Having a `maxLiquidationDiscount = 0` means that the liquidator won't receive any bonus on the debt repaid on behalf of the violator.